### DISCUSSION TITLE
How to *remove* bash history completion (macOS)?

### DISCUSSION THREAD

**Discussion by woj-tek** [skim-rs/skim#649] [Shell integrations]
So I installed skim/sk with homebrew and upon first run it prompted to "install support", I agreed without paying much attention and now it overtook ctrl+r (bash history reverse search). While it search features are awesome it doesn't clear terminal well so I wanted to remove this integration and for everything holly I can't find a place where it got installed... I checked all bash related files/directories in `~/` and I still don't know where it could have installed itself. Any hint how to remove that integration would be welcomed

(it would be better as "discussion" but I don't see those enabled in this repo)

**Comment by LoricAndre** [comment]
Which shell are you using ? Homebrew probably modified your `~/.bashrc` or `~/.zshrc` to add a line sourcing one of the shell keybinding files, which you can delete to disable those keys.

**Comment by woj-tek** [comment]
I'm using bash (from homebrew) and that's the thing - there is nothing in `~/.bashrc` :/

**Comment by LoricAndre** [comment]
Is this useful ? I'm not a homebrew user but this looks like it could help https://formulae.brew.sh/formula/bash-completion

**Comment by mobsenpai** [comment]
@woj-tek Did you mean this #565 . Cuz if so I also have this issue, but I am on nixos.
